src/libs/API.js

```diff
@@ -105,7 +105,30 @@ function makeRequestWithSideEffects(command, apiCommandParameters = {}, onyxData
  * @param {Object} [onyxData.failureData] - Onyx instructions that will be passed to Onyx.update() when the response has jsonCode !== 200.
  */
 function read(command, apiCommandParameters, onyxData) {
-    makeRequestWithSideEffects(command, apiCommandParameters, onyxData, CONST.API_REQUEST_TYPE.READ);
+    SequentialQueue.flush().then((isQueueRunning) => {
```

Comment:
What is `isQueueRunning`? It doesn't look like anything is being passed to the `resolve()` so won't this always be `undefined`?

Reply:
Oh wait, nevermind, I see that there is a second call to `resolve()` at the top.